zerver/lib/test_helpers.py

```diff
@@ -141,6 +142,17 @@ def cursor_executemany(self, sql, params=()):
     TimeTrackingCursor.execute = old_execute # type: ignore # https://github.com/JukkaL/mypy/issues/1167
     TimeTrackingCursor.executemany = old_executemany # type: ignore # https://github.com/JukkaL/mypy/issues/1167
 
+def get_test_image_file(filename):
+    # type: (str) -> IO[Any]
+    test_avatar_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../tests/images'))
+    return open(os.path.join(test_avatar_dir, filename), 'rb')
+
+def avatar_disk_path(user_profile, medium=False):
+    # type: (UserProfile, bool) -> str
+    avatar_url_path = avatar_url(user_profile, medium)
+    avatar_disk_path = os.path.join(settings.LOCAL_UPLOADS_DIR, "avatars",
+                                    avatar_url_path.split("/")[-1].split("?")[0])
```

Comment:
This also works if you are sure that you need only the last part of the url path.